Cookbook/README.md

```diff
@@ -39,6 +39,7 @@ iOS Cookbook 👩‍🍳
 * [Outstanding Technical Debt and Legacy Code](./Technical-Documents/TechnicalDebt.md)
     * [BBAddAdditionalPatientInformationViewControllerV2](./Technical-Documents/BBAddAdditionalPatientInformationViewControllerV2.md)
 
-* [✏️] What is Gherkin (the When / Given / Then/ And formatting used by QA) and how can i determine it's correctness in QA PRs
+* ✏️ What is Gherkin (the When / Given / Then/ And formatting used by QA) and how can i determine it's correctness in QA PRs
```

Comment:
```suggestion
* ✏️ What is Gherkin (the When / Given / Then/ And formatting used by QA) and how can i determine its correctness in QA PRs
```

Actually this was a typo I made since I wrote that line 😅